go/fn/resourcelist.go

```diff
@@ -88,20 +88,24 @@ func ParseResourceList(in []byte) (*ResourceList, error) {
 	}
 	if found {
 		rl.FunctionConfig = asKubeObject(fc)
+	} else {
+		rl.FunctionConfig = DummyKubeObject()
 	}
 
-	items, _, err := rlObj.obj.GetNestedSlice("items")
+	items, found, err := rlObj.obj.GetNestedSlice("items")
```

Comment:
This seems to be a bug that does not accept empty `resourcelist.items`. I think this is invalid for generator.

Reply:
Good point!
In the generator use case, it's likely `items` is empty. But `functionConfig` may be populated. We should not immediately return rl if not found `items in line 100 below.